aws/apis/network/v1alpha2/vpc_types.go

```diff
@@ -22,10 +22,10 @@ import (
 	runtimev1alpha1 "github.com/crossplaneio/crossplane-runtime/apis/core/v1alpha1"
 )
 
-// VPCParameters defines the desired state of a VPC
+// VPCParameters define the desired state of an AWS Virtual Private Cloud.
 type VPCParameters struct {
-	// CIDRBlock is the IPv4 network range for the VPC, in CIDR notation. For example, 10.0.0.0/16.
-	// +kubebuilder:validation:Required
+	// CIDRBlock is the IPv4 network range for the VPC, in CIDR notation. For
+	// example, 10.0.0.0/16. +kubebuilder:validation:Required
```

Comment:
this looks like the kubebuilder validation marker accidentally made it's way into a comment line.  I think that breaks it because later on I see it verbatim in the CRD's yaml.

Reply:
Fixed